packages/shared/src/components/squads/SquadPostsProgressBar.tsx

```diff
@@ -42,7 +42,7 @@ export const SquadPostsProgressBar = ({
           Posts
         </span>
         <span className={classNames(className?.labelContent ?? 'font-bold')}>
-          {effectivePostsCount} / {goal}
+          {effectivePostsCount}/{goal}
```

Comment:
This looks better to me :)

Reply:
Awesome. Let's move forward without spaces.